sagemaker_studio_image_build/cli.py

```diff
@@ -70,6 +78,15 @@ def main():
         "--repository",
         help="The ECR repository:tag for the image (default: sagemaker-studio-${domain_id}:latest)",
     )
+    build_parser.add_argument(
+        "--image",
+        help="The ECR repository:tag for the image (default: sagemaker-studio-${domain_id}:latest)",
+    )
```

Comment:
This looks identical to the `--repository` parameter. What is the use for this?
```suggestion
```

Reply:
I was also going to add support for targeting a different [CodeBuild image](https://docs.aws.amazon.com/codebuild/latest/userguide/build-env-ref-available.html) in addition to compute type.  But decided to keep this PR just to compute type - so I will remove this option.